.github/workflows/assets-validation.yaml

```diff
@@ -27,6 +27,10 @@ defaults:
   run:
     shell: bash
 
+permissions:
+  # Required for OIDC login to Azure
+  id-token: write
+
 jobs:
   validate:
     name: Validate
```

Comment:
below the line "runs-on: ubuntu-latest", add this line which should solve the access issue
```
    environment: Testing
```

Reply:
Thanks @lykelly19